routes/article.js

```diff
@@ -94,12 +94,20 @@ function updateNews() {
     })
 }
 
+function getAmount(vote) {
+    return Math.abs(vote) / vote || 0
+}
+
 router.post('/vote', (req, res) => {
     console.log(`votes modified by ${req.body.vote} for ${req.body.id}`)
     console.log(req.body)
-    let amount = parseInt(req.body.vote, 10)
+    let amount = getAmount(req.body.vote)
+    console.log(amount)
+    if (amount === 0) {
+        return false
+    }
     Article.findById(req.body.id).then((found) => {
-        if (!found.length) {
+        if (found) {
```

Comment:
Find a better name for the article found such as `article`. "Found" gets confusing since it doesn't tell you anything about what it is